cloud/deploys/bin/azure/bastion

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env bash
```

Comment:
nit: to follow the style guide, library shell files should have `.sh` file extension